File: ui/src/models/Question.ts
import Answer from './Answer';

export default class Question {
  public _id?: string;
  public question: string;
  public channels: string[];
  public askTime?: Date;
  public answers?: Answer[];
  // public from: { type: string, value: string };
  // public to: Array<{ type: string, value: string }>;
}

File: job_system/src/entities/Question.java
package entities;

import java.util.Date;
import java.util.List;

public class Question {

	public String text;
	public Date askTime;
	public List<String> channels;
	
	public Question(String text, Date time, List<String> channels) {
		
		this.text = text;
		this.askTime = time;
		this.channels = channels;
	}
}
